### ISSUE TITLE
conda_forge activity doesn't check for conda-smithy

### ISSUE THREAD

**Issue by dougalsutherland** [regro/rever#171]
Just did this in a conda install where I didn't have conda-smithy, and got:

```
Rerendering the feedstock

CommandNotFoundError: No command 'conda smithy'.

Counting objects: 4, done.
Delta compression using up to 12 threads.
```

and it happily proceeded and opened the PR (https://github.com/conda-forge/arxiv-collector-feedstock/pull/1).

It should probably instead ask the bot to rerender in the PR if you don't have conda-smithy installed. :)

**Comment by scopatz** [comment]
Good point! This is now fixed in https://github.com/regro/rever/pull/174
